client/components/editor.vue

```diff
@@ -63,6 +63,8 @@ import { AtomSpinner } from 'epic-spinners'
 import { Base64 } from 'js-base64'
 import { StatusIndicator } from 'vue-status-indicator'
 
+import renderersQuery from 'gql/admin/rendering/rendering-query-renderers.gql'
```

Comment:
I think it is not right to call admin query here but duplicating the same query in editor section also do not feel right. Any suggestions how to proceed?